### ISSUE TITLE
[MU4 Issue] Enabling high-contrast switches to dark theme and swaps theme icons

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9220]
**Describe the bug**
From Jessica Williamson : Just realized something about the theme placements between normal mode and high contrast mode. On normal we have light mode on left, dark on right. When high contrast mode is on, dark version is now on left, white on right. Wouldn't it make sense if the high contrast ones were swapped to match the normal ones? so both light versions on left and darks on right?

**To Reproduce**
1. Go to Edit>Preferences>Appearance
2. Check 'Enable high-contrast' (see video below)

**Expected behavior**
It should not switch from Light to Dark theme and should not swap the icons

**Screenshots**

![Enable high-contrast](https://user-images.githubusercontent.com/90187801/134334184-f8ea3d67-1813-40be-82e0-eecaa5f5db5d.gif)

**Desktop:**
 OS: - Win10/Mac10.15.7/Linux Mint 20.2

**Comment by DmitryArefiev** [comment]
Since, that option (in Welcome to MuseScore 4 Getting started dialog) will be one of the first things that user see, adding to Alpha release (with P3)

https://user-images.githubusercontent.com/90187801/143444029-0836b7b9-6509-47b6-9712-97e3f4fa833b.mp4

**Comment by nanu1605** [comment]
@DmitryArefiev @RomanPudashkin @vpereverzev  Hi, I am pretty new to open source but would love to work on this issue. Could you please help me with the issue more.

**Comment by cbjeukendrup** [comment]
These theme icons are ultimately displayed in ThemeSamplesList.qml. Maybe you can track down yourself where the data for that component comes from.
<details><summary>Hint</summary>
Search in the code where ThemeSamplesList is used, and look how the data is injected into the component.
<details><summary>One more hint</summary>
You will find that they are used in ThemesPage.qml (for the "Welcome to MuseScore 4 Getting started" dialog) and ThemesSection.qml (for Preferences). The data comes in trough the "themes" property.
Now, look upwards in the hierarchy to find out from which C++ model the value for that property ultimately comes.
<details><summary>One more hint</summary>
You will find that the data comes from ThemesPageModel (for the "Welcome to MuseScore 4 Getting started" dialog) and AppearancePreferencesModel (for Preferences). In those models, look how the relevant methods work.
<details><summary>One more hint</summary>
Ultimately, you will find that the order of the themes depends on the order in allStandardThemeCodes() in uitypes.h.
</details>

</details>

</details>

</details>

**Comment by DmitryArefiev** [comment]
Tested on master (Win10/Mac12) - THE ISSUE STILL OCCURS when enabling high-contrast in Dark theme

https://user-images.githubusercontent.com/90187801/149755896-c940c0b7-1e0b-4f23-848d-64d97921c9ab.mp4

Reopening

**Comment by jithinjohn2k2** [comment]
https://user-images.githubusercontent.com/73242397/149920094-ad2ddcec-6021-4cd5-b1cb-db7f9d68bfd1.mp4

@DmitryArefiev Is this the expected behavior that will close this issue?

**Comment by DmitryArefiev** [comment]
@jithinjohn2k2 Yes, it should not change a theme when enabling/disabling high-contrast.

Thanks!

**Comment by DmitryArefiev** [comment]
Tested on Win10, Mac12, LinuxMint 20.2 - FIXED in #10408
